src/controllers/import.js

```diff
@@ -83,12 +83,43 @@ function ImportController(context) {
     }
   }
 
+  function validateIsoDates(startDate, endDate) {
+    if (!startDate || !endDate) {
```

Comment:
May need to update the import to include `hasText`, too.

```suggestion
    if (!hasText(startDate) || !hasText(endDate)) {
```

Reply:
is this check really necessary? the `isIsoDate` will do the job.